gradio/blocks.py

```diff
@@ -1591,7 +1588,7 @@ def get_time():
                 demo.load(get_time, inputs=None, outputs=dt)
             demo.launch()
         """
-        if isinstance(self_or_cls, type):
+        if self is None:
```

Comment:
This changes the behavior a little bit --> previously, if you tried to do:

```
gr.Blocks.load("huggingface/t5-base")
```

we would raise a more informative error: `ValueError: Blocks.load() requires passing parameters as keyword arguments`. Now the error is just: "AttributeError: 'str' object has no attribute 'set_event_trigger'"

But I think that's okay since this has been soft deprecated for a while.